aliases/available/git.aliases.bash

```diff
@@ -66,6 +66,11 @@ alias gnew="git log HEAD@{1}..HEAD@{0}"
 # Add uncommitted and unstaged changes to the last commit
 alias gcaa="git commit -a --amend -C HEAD"
 alias ggui="git gui"
+alias gcam="git commit -am"
+alias gcsam="git commit -S -am"
+alias gmu="git fetch origin -v && git merge origin/master"
```

Comment:
The `gmu` alias is already defined here in the same file: https://github.com/Bash-it/bash-it/blob/master/aliases/available/git.aliases.bash#L53